fastchat/eval/eval_gpt_review.py

```diff
@@ -44,62 +50,84 @@ def parse_score(review):
         return [-1, -1]
 
 
+def gen_prompt(reviewer_jsons, prompt_jsons, cat, ques, ans1, ans2):
+    reviewer_idx = 0
+    for idx, reviewer in enumerate(reviewer_jsons):
```

Comment:
how we handle the case that there are no matching categories? In this case we should fallback to `general`

Reply:
Yes by default `reviewer_idx = 0` which indicates `general` category is used. I added a comment to clarify it.